tests/suite/system_suite_test.go

```diff
@@ -73,6 +73,7 @@ var (
 	localChartPath    string
 	address           string
 	version           string
+	chartVersion      string
```

Comment:
Where does this get initialized?

Reply:
it doesn't, because there's not a way to say "the latest chart" so when it's empty we don't use it.

Reply:
I wonder if we just use a literal empty string for those cases. Makes it more obvious so someone won't have the same question as me.